### PR TITLE
Plugin documentation

### PR DESCRIPTION
**Pull request by bobmulder**
Removed the plugin-assets.rst because we have the plugin-shell.rst.
Also I fixed the code of the assets-section because this was outdated.

### CODE REVIEW

**Comment by antograssiot** [comment]
@bobmulder If you delete this file you must remove it from the toc in the bottom of en/console-and-shells.rst or it might cause build error.

File: en/console-and-shells/plugin-shell.rst [hunk 1]
```diff
@@ -47,3 +47,25 @@ This will remove the line ``Plugin::load('MyPlugin',...`` from your
 .. meta::
     :title lang=en: Plugin Shell
     :keywords lang=en: api docs,shell,plugin,load,unload
+
```
Comment (antograssiot) [inline en/console-and-shells/plugin-shell.rst]: this content must be before the '.. meta ::'

Comment (bobmulder) [inline en/console-and-shells/plugin-shell.rst]: Thanks @antograssiot, I will change that stuff tomorrow.

**Comment by bobmulder** [comment]
@antograssiot Fixed

**Comment by antograssiot** [comment]
thank's @bobmulder

**Pull request merged**